docs/dev-guide/migrations/v3-to-v4.md

```diff
@@ -0,0 +1,28 @@
+# Migrate from v3 to v4
+
+To migrate from v3 to v4 `manually`, you have to add two new settings to the portal web app. These two settings are required to to pull devices telemetry from the IoT Hub:
+
+| Name | Setting Type | Detail |
+|---|---|---|
+| `IoTHub__EventHub__ConsumerGroup` | Application setting | (Default value `iothub-portal`) The name of the consumer group used to to pull data from the IoT Hub |
+| `IoTHub__EventHub__Endpoint` | Connection string  | The IotHub Event Hub compatible endpoint |
+
+Below the required steps for each settings:
+
+## `IoTHub__EventHub__ConsumerGroup`
+
+1. Go to your IoT Hub
+2. Navigate to menu Built-in endpoints
+3. Create a consumer group with the name `iothub-portal`
+    ![iothub-create-consumer-group.png](/images/iothub-create-consumer-group.png)
+4. Back to the portal web app, add a new application setting with name `IoTHub__EventHub__ConsumerGroup` and with value `iothub-portal`
+
+## `IoTHub__EventHub__Endpoint`
+
+1. Go to your IoT Hub
+2. Navigate to menu Built-in endpoints
+3. On the section Event Hub compatible endpoint
+      1. Select the shared access policy `service`
```

Comment:
Here we can add an annotation saying that users can create their own Shared access policies. But the portal needs at least the ``Service Connect`` permission.